.travis.yml

```diff
@@ -1,4 +1,20 @@
 language: generic
+matrix:
+  include:
+    - name: "generic tests"
+    - name: "flake8 lint on Python 2.7"
+      language: python
+      python: 2.7
+      install: pip install flake8
+      script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Couldn't these be checked with the existing PEP8 script?

Reply:
1. __pep8__ was deprecated a long time ago in favor of __pycodestyle__.
2. __flake8__ is a superset of __pycodestyle__ so the Fxxx errors are in __pycodestyle__ but the Exxx errors are not.
3. __flake8__ provides different results when run on Python 2 vs. Python 3.

Reply:
Ah, I see. Thanks for the explanation.
In that case, it would be better to update [the existing script](https://github.com/iovisor/bcc/blob/master/scripts/py-style-check.sh) rather than add new separate tests.